src/transactions/transferFeedUtils.ts

```diff
@@ -330,6 +333,9 @@ export function useTransferFeedDetails(transfer: FeedTokenTransfer) {
       } else if (providerInfo) {
         title = t('feedItemReceivedTitle', { displayName })
         subtitle = t('tokenDeposit', { token: tokenInfo?.symbol ?? '' })
+      } else if (isCoinbasePaySender) {
+        title = 'Deposit'
```

Comment:
Should this be a translated string?

Reply:
Yes! Left it as a placeholder and didn't update the translation file.